desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -37,7 +35,11 @@ TabDiveInformation::TabDiveInformation(QWidget *parent) : TabBase(parent), ui(ne
 	connect(ui->diveType, SIGNAL(currentIndexChanged(int)), this, SLOT(diveModeChanged(int)));
 	QString CSSSetSmallLabel = "QLabel { color: mediumblue; font-size: " +                        /* // Using label height ... */
 		QString::number((int)(0.5 + ui->diveHeadingLabel->geometry().height() * 0.66)) + "px;}"; // .. set CSS font size of star widget subscripts
+#if defined(Q_OS_WIN)
 	ui->scrollAreaWidgetContents_3->setStyleSheet("QGroupBox::title { color: mediumblue;} ");
+#else
+	ui->scrollAreaWidgetContents_3->setStyleSheet("QGroupBox { border: 1px solid silver; border-radius: 4px; margin-top: 0.65em; background-color: gainsboro;} QGroupBox::title { color: mediumblue;} ");
```

Comment:
I think this deserves a comment why the different code for Windows is necessary

Reply:
This commit with respect to the overwrite indication is ready for merging. 

The OS-specific CSS styling is an attempt to get the Information tab to look fairly similar in the three platforms we support. With the above code the Win rendering is:
![Win](https://user-images.githubusercontent.com/5078706/69333509-68edd700-0c61-11ea-9ec3-81781b0ec79d.jpg)
And the Linux rendering is:
![Linux](https://user-images.githubusercontent.com/5078706/69333521-72773f00-0c61-11ea-910d-fb4ca7bf7f8b.jpg)
which are roughly comparable. With the previous code the Mac rendering was very unfriendly but I have no idea what it looks like on a Mac now. Assuming that Qt on a Mac and on Linux are fairly similar (both Unix-derived) I hope the Mac rendering will also be much more friendly now

Reply:
Of course I will do more changes after @glance and @bstoeger have inspected the code.